### PR TITLE
pop open console when deploying

### LINKED ISSUE #979: autoopen console on deploy and bring to front
Just maybe this already works, but find out.

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis fix #979

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java [hunk 1]
```diff
@@ -142,6 +148,20 @@ public void done(IJobChangeEvent event) {
       }
     });
     deploy.schedule();
+    
+    // https://wiki.eclipse.org/FAQ_How_do_I_find_the_active_workbench_page%3F
+    IWorkbenchWindow window = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java]: I've just updated that wiki page as this `PlatformUI.getWorkbench().getActiveWorkbenchWindow()` is actually an anti-pattern!

We have the `ExecutionEvent`, so just use:
```
IWorkbenchWindow window = HandlerUtil.getActiveWorkbenchWindow(event);
```

**Comment by akerekes** [comment]
@briandealwis @elharo in MessageConsoleUtilities on line 76 we use `consoleManager.showConsoleView(messageConsole);` for the same effect. Are these equivalent? If so, I'd rather use `consoleManager` as it's a much shorter solution, and add it the the `createConsole` method that is used to create the DeployConsole in `StandardDeployCommandHandler`.

**Comment by briandealwis** [comment]
`ConsoleManager#showConsoleView()` is almost equivalent — its implementation uses the `PlatformUI.getWorkbench().getActiveWorkbenchWindow()` (namely as `ConsoleManager` doesn't have any context to the appropriate window).

But in the interest of reducing code to maintain, I support its use!

**Comment by elharo** [comment]
switched to ConsoleManager#showConsoleView()

**Pull request merged**